.eslintrc.json

```diff
@@ -11,27 +12,71 @@
 		"browser": true,
 		"node": true
 	},
+	"globals": { "Proxy": "readonly" },
+	"extends": [
+		"eslint:recommended",
+		"plugin:@typescript-eslint/recommended",
+		"plugin:@stylistic/disable-legacy"
+	],
+	"plugins": ["@typescript-eslint", "@stylistic"],
 	"rules": {
-		"semi": 1,
+		"@stylistic/arrow-spacing": 1,
```

Comment:
Could we use [a config format that allows comments](https://eslint.org/docs/latest/use/configure/configuration-files#configuration-file-formats) so we can document what each of these is? (at least those not entirely obvious from the name, which is most of them)

That will also help me review the rest of this :)